indexer/src/api/v1/payments.rs

```diff
@@ -11,7 +11,9 @@ use indexer_db::messages::payment::{
     PaymentByReceiverPartition, PaymentBySenderPartition, TxIdToPaymentPartition,
```

Comment:
not sure why, but in this file specifically, the formatter isn't working for me, i've also try calling clippy manually (CLI)